sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -2,6 +2,12 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
+# pylint: skip-file
```

Comment:
do we need to add these in  ...n/azure-communication-administration/azure/communication/administration/_shared/models.py?